templates/etc/icinga2/conf.d/downtimes.conf.j2

```diff
@@ -6,19 +6,17 @@
  */
 
 {% for downtime in icinga2_master_downtimes %}
-apply ScheduledDowntime "downtime-{{ downtime.name }}" to Service {
+apply ScheduledDowntime "downtime-{{ downtime.name }}" to {{ 'Service' if downtime.services else 'Host' }} {
```

Comment:
As discussed with @ayekat . According to [this forum post](https://community.icinga.com/t/recuring-downtime-for-host-and-its-services/779/3) one apply rule isn't sufficient to catch the host alive check **and** the host's services at once. Two apply rules are needed.